static/app/gettingStartedDocs/javascript/nuxt.tsx

```diff
@@ -0,0 +1,371 @@
+      ),
+      configurations: getInstallConfig(),
+    },
+  ],
+  configure: (params: Params) => [
+    {
+      type: StepType.CONFIGURE,
+      configurations: [
+        {
+          description: tct(
+            'Add the Sentry Nuxt module in your [code:nuxt.config.ts] file:',
+            {code: <code />}
+          ),
+          code: [
+            {
+              label: 'TypeScript',
+              value: 'typescript',
+              language: 'typescript',
+              code: getNuxtModuleSnippet(),
+            },
+          ],
+        },
+        {
+          description: tct(
+            'For the client, create a [codeFile:sentry.client.config.ts] file in your project root and initialize the Sentry SDK:',
+            {codeFile: <code />}
+          ),
+          code: [
+            {
+              label: 'TypeScript',
+              value: 'typescript',
+              language: 'typescript',
+              code: getSdkClientSetupSnippet(params),
+            },
+          ],
+        },
+        {
+          description: (
+            <Fragment>
+              {tct(
+                'For the server, create a [codeFile:sentry.server.config.ts] file in your project root and initialize the Sentry SDK:',
+                {codeFile: <code />}
+              )}
+
+              <StyledAlert type="info" showIcon>
+                {tct(
+                  'To complete the server-side setup, follow the [link:Sentry Nuxt docs] for guidance. Nuxt compiles your code in ESM on the server side as well, so the deployment setup can get tricky depending on where you deploy your application.',
+                  {
+                    link: (
+                      <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/nuxt/#server-side-setup" />
+                    ),
+                  }
+                )}
+              </StyledAlert>
+            </Fragment>
+          ),
+          code: [
+            {
+              label: 'TypeScript',
+              value: 'typescript',
+              language: 'typescript',
+              code: getSdkServerSetupSnippet(params),
+            },
+          ],
+        },
+      ],
+    },
+    {
+      title: t('Upload Source Maps'),
+      description: tct(
+        'To upload source maps to Sentry, follow the [link:instructions in our documentation].',
+        {
+          link: (
+            <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/nuxt/#add-readable-stack-traces-to-errors" />
+          ),
+        }
+      ),
+    },
+  ],
+  verify: () => [
+    {
+      type: StepType.VERIFY,
+      description: t(
+        "This snippet contains an intentional error and can be used as a test to make sure that everything's working as expected."
+      ),
+      configurations: [
+        {
+          code: [
+            {
+              label: 'JavaScript',
+              value: 'javascript',
+              language: 'html',
```

Comment:
```suggestion
              language: 'javascript',
```

Reply:
The code is Vue code so I have to use html here to get correct syntax highlighting. But I changed the label and value to vue.